app/src/main/java/com/developersbreach/composeactors/ui/screens/movieDetail/MovieDetailsUI.kt

```diff
@@ -84,44 +101,96 @@ fun MovieDetailsUI(
             )
         },
     ) {
-        Box(
-            modifier = Modifier
-                .fillMaxSize()
-                .testTag("TestTag:MovieDetailScreen")
-        ) {
-            // Background poster with layer reveal effect
-            LayerRevealImage(uiState.movieData?.poster, isLayerRevealAnimationEnded)
-            // Fade enter animation detail screen once layer reveal completes
-            if (isLayerRevealAnimationEnded.value) {
-                AnimatedVisibility(
-                    visibleState = state,
-                    enter = fadeIn()
-                ) {
-                    // Main details content
-                    MovieDetailsContent(
-                        modifier = modifier,
-                        uiState = uiState,
-                        navigateUp = navigateUp,
-                        showFab = showFab,
-                        openMovieDetailsBottomSheet = openMovieDetailsBottomSheet,
-                        selectBottomSheetCallback = selectBottomSheetCallback
-                    )
-                }
+        val scaffoldState = rememberBottomSheetScaffoldState()
```

Comment:
Should be moved to the top declaration inside this function.

Also we can use better naming for `animatedSheetPeekHeight`, because we have two different sheets here, scaffold and modal so we wouldn't know which one this belongs to. May be `animatedScaffoldSheetPeekHeight` ?